flink-runtime/src/test/java/org/apache/flink/runtime/jobgraph/JobTaskVertexTest.java

```diff
@@ -84,10 +88,10 @@ public void testConnectMultipleTargets() {
 	@Test
 	public void testOutputFormatVertex() {
 		try {
-			final TestingOutputFormat outputFormat = new TestingOutputFormat();
+			final OutputFormat outputFormat = new TestingOutputFormat();
 			final OutputFormatVertex of = new OutputFormatVertex("Name");
 			new TaskConfig(of.getConfiguration()).setStubWrapper(new UserCodeObjectWrapper<OutputFormat<?>>(outputFormat));
-			final ClassLoader cl = getClass().getClassLoader();
+			final ClassLoader cl = new FlinkUserCodeClassLoader(new URL[0], getClass().getClassLoader());
```

Comment:
This doesn't exist anymore but you can use `FlinkUserCodeClassLoaders.childFirst()`/`parentFirst()`.